keel-core/src/main/kotlin/com/netflix/spinnaker/keel/caffeine/CacheFactory.kt

```diff
@@ -71,6 +70,12 @@ class CacheFactory(
     CaffeineCacheMetrics.monitor(meterRegistry, this, cacheName)
 }
 
+fun <K, V> (suspend (K) -> V?).toAsyncCacheLoader() : AsyncCacheLoader<K, V> =
```

Comment:
I look at this code and think "I have no idea if it's correct or not, but I bet if it compiles then it'll work properly."